website/docs/developer/research/research-localstack.md

```diff
@@ -161,12 +171,16 @@ Install the AWS CLI to run CLI commands in the docker container.
     pip install --trusted-host pypi.org --trusted-host files.pythonhosted.org awscli-local
 
 This prevents the need for specifying --endpoint-url everytime aws cli command is run. The container is now ready to test AWS services locally.
-
+:::Note
+AWSlocal is a thin wrapper around the aws command line interface for use with LocalStack.
+:::
 ### Examples of AWS services deployed using localstack
 Some examples of CLI commands and their corresponding outputs are shown below.
 #### API gateway
-  Command: `awslocal apigateway create-rest-api --name test-api`
-
+  ```commandline
```

Comment:
I would combine these into a single commandline block instead of a bunch of separate ones and add some comments for users. For example for this section I would maybe do something like this.

```
```commandline

# Create the api gateway called test-api
C:\Users\test\localstack-test> awslocal apigateway create-rest-api --name test-api

Response: 
        {
            "id": "0zqyew47ie",
            "name": "test-api",
            "createdDate": 1623862127,
            "version": "V1",
            "binaryMediaTypes": [],
            "apiKeySource": "HEADER",
            "endpointConfiguration": {
                "types": [
                    "EDGE"
                ]
            },
            "tags": {},
            "disableExecuteApiEndpoint": false
        }

# Get the parent resource id to create an api gateway resource using the api gateway id from above
C:\Users\test\localstack-test> PARENT_RESOURCE_ID=$(awslocal apigateway get-resources --rest-api-id 0zqyew47ie --query 'items[?path==`/`].id' --output text)

# Create an APRI resource Gateway
C:\Users\test\localstack-test> awslocal apigateway create-resource --rest-api-id 0zqyew47ie --parent-id $PARENT_RESOURCE_ID --path-part "{test}"

Response: 
    {
        "id": "7z17qim7fw",
        "parentId": "d22pls3twh",
        "pathPart": "{test}",
        "path": "/{test}"
    }

```